src/app/components/PriceChart.tsx

```diff
@@ -260,12 +262,75 @@ function PriceChartCanvas(props: PriceChartProps) {
   );
 }
 
+enum ChartTabOptions {
+  TRADING_CHART = "TRADING_CHART",
+  PAIR_INFO = "PAIR_INFO",
+}
+
 export function PriceChart() {
```

Comment:
Can you rename this component to `TradingChartOrPairInfo` for clarity? Otherwise its a bit confusing to me what the difference between priceChart and TradingChart is. 

You will also need to adapt the naming inside the trade page where this component is imported and rendered.